script.js

```diff
@@ -33,3 +34,165 @@ window.onresize = () => {
+    document.querySelector(`.project_${index}`).appendChild(list);
+  });
+
+  if (index % 2 === 0) {
+    const header = document.querySelector(`#header_${index}`);
+    const cardBody = document.querySelector(`.cb_${index}`);
+    header.classList.toggle('flip1');
+    cardBody.classList.toggle('flip2');
+  }
+}
+
+navLinks.forEach((link) => link.addEventListener('click', remove));
+
+window.onload = () => {
+  const projects = [
+    {
+      name: 'Library',
+      description:
+        'A daily selection of privately personalized reads sit amet consectetur adipisicing elit.',
+      featureImage: 'portfolio1.png',
+      techStack: ['html', 'css', 'javascript'],
+      liveLink: 'https://sawmon71293.github.io/personal-portfolio/',
+      sourceLink: 'https://github.com/sawmon71293/personal-portfolio',
+    },
+    {
+      name: 'Job Agency',
+      description:
+        'A daily selection of privately personalized readsr sit amet consectetur adipisicing elit.',
+      featureImage: 'portfolio2.png',
+      techStack: ['html', 'css', 'javascript'],
+      liveLink: 'https://sawmon71293.github.io/personal-portfolio/',
+      sourceLink: 'https://github.com/sawmon71293/personal-portfolio',
+    },
+    {
+      name: 'Songs Library',
+      description:
+        'A daily selection of privately personalized reads dolor sit amet consectetur adipisicing elit. ',
+      featureImage: 'portfolio3.png',
+      techStack: ['html', 'react', 'Ruby on Rails'],
+      liveLink: 'https://sawmon71293.github.io/personal-portfolio/',
+      sourceLink: 'https://github.com/sawmon71293/personal-portfolio',
+    },
+    {
+      name: 'Portal',
+      description:
+        'A daily selection of privately personalized reads sit amet consectetur adipisicing elit. ',
+      featureImage: 'portfolio4.png',
+      techStack: ['html', 'react', 'Ruby on Rails'],
+      liveLink: 'https://sawmon71293.github.io/personal-portfolio/',
+      sourceLink: 'https://github.com/sawmon71293/personal-portfolio',
+    },
+  ];
+  projects.forEach((project, index) => {
+    createCard(project, index);
+  });
+
+  const openModalButtons = document.querySelectorAll('[data-modal-target]');
+
+  openModalButtons.forEach((button) => {
+    button.addEventListener('click', () => {
+      const modal = document.querySelector(button.dataset.modalTarget);
+      const project = JSON.parse(decodeURIComponent(button.dataset.project));
+
+      const projectModalTemplate = `
+      
+         <header class="modal-header">
+          <div><h2 id="modal-title">${project.name}</h2></div>
+          <button data-close-button class="close-button">&times;</button>
+         </header>
+          <div class="wrap">
+         <div class="modal-body">
+          <ul class="modal-exp">
+            <li class="canopy">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <img class="portfolio" src="/img/${project.featureImage}" alt="Portfolio">
```

Comment:
Please consider reviewing your design as it's not in accordance with the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=fkLOCTDtdpHXqeN9-0). Currently, when a user clicks on the popup he is not able to see the project image. The possible reason for it can be due to absolute and relative paths. I am attaching the screenshots of both of the designs for better understanding:- 

**Figma Design:-** 

![image](https://user-images.githubusercontent.com/94223367/217605704-73eb2039-158c-4e47-80c4-30b2c1d78c4b.png)


**Your Design:-**

![image](https://user-images.githubusercontent.com/94223367/217605734-5ae19d68-8b5d-4dc4-b32a-061ffc3a83b6.png)

**How you can resolve it:-**  _It is quite possible that it's working at your place but not here. To review it and resolve to consider using the following code instead of it it might help you:-_

`<img class="portfolio" src="./img/${project.featureImage}" alt="Portfolio">`


**REMEMBER:-** _As it's not working properly so I am unable to review the other designs properly consider reviewing other styles for popup also._

Reply:
Current version has not been yet merged it. That's why readme is outdated. Thanks for your detailed review.